### ISSUE TITLE
Compatibility issue with phpBB 3.3.0

### ISSUE THREAD

**Issue by luilegeant** [marttiphpbb/phpbb-ext-calendarmonthview#2]
## Description:
Installing this extension as well as the 7 others (list bellow) doesn't work, the page stays blank (after enabling the injection of a link in the overallheader section.

## How to reproduce:
1. Setup a phpBB board version 3.3.0.
2. install and enable the following extensions:
  - [phpextcalendar](https://github.com/marttiphpbb/phpbb-ext-phpextcalendar/releases/download/0.1.0/phpextcalendar_0_1_0.zip)
  - [calendarmono](https://github.com/marttiphpbb/phpbb-ext-calendarmono.git)
  - [jqueryuidatepicker](https://github.com/marttiphpbb/phpbb-ext-jqueryuidatepicker.git)
  - [calendarmonoinput](https://github.com/marttiphpbb/phpbb-ext-calendarmonoinput.git)
  - [calendartag](https://github.com/marttiphpbb/phpbb-ext-calendartag.git)
  - [topicprefixtags](https://github.com/marttiphpbb/phpbb-ext-topicprefixtags.git)
  - [menuitems](https://github.com/marttiphpbb/phpbb-ext-menuitems.git)
3. install this extension
4. in the configuration for this extension: Links section, check the first box `Overall header navigation prepend`
5. navigate to the home page of the board the page stays blank

The server logs provided me this:
```
[error] 6#6: *93 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught InvalidArgumentException: The routing file "/var/www/forum/ext/marttiphpbb/calendarmonthview/config/routing.yml" contains unsupported keys for "marttiphpbb_calendarmonthview_page_controller": "pattern". Expected one of: "resource", "type", "prefix", "path", "host", "schemes", "methods", "defaults", "requirements", "options", "condition", "controller". in /var/www/forum/vendor/symfony/routing/Loader/YamlFileLoader.php:206
```

## How to fix:
Change the 2 following files:
1. `./config/routing.yml` replace the keyword the `pattern` by `path`
2. `./ext.php` change the minimal supported version from `3.2.5` to `3.3.0` => this one is a guess, I didn't check the validity with previous version.

**Comment by marttiphpbb** [comment]
Twig and Symfony got updated. That's why. I'll have a look into it.

**Comment by marttiphpbb** [comment]
Fixed in version 0.8.0
